src/utils/randomNumberGenerator.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const randomNumberGenerator = () => {
+  const uniqueNumbers = [];
+
+  while (uniqueNumbers.length < 4) {
```

Comment:
let's create variable for `4` and reuse in different places

Reply:
Thanks, now it looks better.